examples/updatecli.generic/yaml-files.yaml

```diff
@@ -0,0 +1,98 @@
+---
+title: Show a set of yaml resources as a generic example
```

Comment:
I am wondering if this file should be moved to the e2e directory, I am considering removing the examples directory. 
My motivation is, we don't test them so we don't detect when they are outdated while those in the e2e directory will fail

Reply:
You've added it here, I don't mind if you want to move it elsewhere ^^

Reply:
@olblak  I'd suggest to move them in another PR so we can finally ship this current one, WDYT?